modules/dcache-vehicles/src/main/java/diskCacheV111/vehicles/JobInfo.java

```diff
@@ -24,36 +22,29 @@
     private String _status;
     private final long _jobId;
 
-
-    public JobInfo(long submitTime, long startTime, String status, int id, String clientName, long clientId) {
+     public JobInfo(long submitTime, long startTime, String status, int id, String clientName, long clientId) {
```

Comment:
Did you nudge the constructor?  It looks like the patch has added a space to its indentation 😕